bodhi/server/buildsys.py

```diff
@@ -506,7 +506,16 @@ def createTag(self, tag: str, **opts):
 
     def deleteTag(self, tagid):
         """Emulate tag deletion."""
```

Comment:
Should we adjust the docstring to specify the tagid can be either a string or an int and the behavior in each case?

Reply:
will do